sormas-backend/src/main/java/de/symeda/sormas/backend/sample/PathogenTestFacadeEjb.java

```diff
@@ -36,17 +36,14 @@
 import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
 
+import de.symeda.sormas.api.sample.*;
```

Comment:
Please check the development environment setup guidelines again - don't use asterisk imports (check and change this for all files)